app/models/manageiq/providers/storage_manager/cinder_manager/refresh_helper_methods.rb

```diff
@@ -0,0 +1,173 @@
+module ManageIQ::Providers::StorageManager::CinderManager::RefreshHelperMethods
```

Comment:
Just try to keep the same pattern as other helper_methods. Will combine together in the next PR if needs.